OG_Format.adoc

```diff
@@ -21,16 +21,16 @@ From there the working group organized regular meetings to validate the evolutio
 ////
 == Authors
```

Comment:
Maybe change to original authors in alphabetical order.

Reply:
Let's do like that. That is fine for me.